dash_docs/chapters/getting_started/index.jl

```diff
@@ -0,0 +1,321 @@
+using Dash
+using DashHtmlComponents
+using DashCoreComponents
+
+app = dash()
+
+
+app.layout = html_div() do
+    html_h1("Dash Layout"),
+    html_blockquote(dcc_markdown("This is the 2nd chapter of the [Dash Tutorial](/).
+    The previous chapter covered [installation](/installation) and the next chapter covers [Dash callbacks](/basic-callbacks).")),
+    dcc_markdown("""
+    This tutorial will walk you through a fundamental aspect of Dash apps, the app `layout`, through several self-contained apps.
+    """),
+    html_hr(),
+    dcc_markdown("""
+    Dash apps are composed of two parts. The first part is the `layout` of the app and it describes what the application looks like.
+    The second part describes the interactivity of the application and will be covered in the [next chapter](/basic-callbacks).
+
+    Dash provides Julia methods for all of the visual components of the application.
+    We maintain a set of components in the `DashCoreComponents` and the `DashHtmlComponents` packages
+    but you can also build your own with JavaScript and React.js
+
+    Note: Throughout this documentation, Julia code examples are meant to be saved as files and executed using `julia app.jl`.
+    These examples are not intented to run in Jupyter notebooks as-is,
+    although most can be modified slightly to function in that environment.
+
+    To get started, create a file called `app.jl` with the following code:
+    """),
+    dcc_markdown("""
+
+    ```
+    using Dash, DashHtmlComponents, DashCoreComponents
+
+
+    app = dash()
+
+    app.layout = html_div() do
+        html_h1("Hello Dash"),
+        html_div("Dash: A web application framework for Julia"),
+        dcc_graph(
+            id = "example-graph",
+            figure = (
+                data = [
+                    (x = ["giraffes", "orangutans", "monkeys"], y = [20, 14, 23], type = "bar", name = "SF"),
+                    (x = ["giraffes", "orangutans", "monkeys"], y = [12, 18, 29], type = "bar", name = "Montréal"),
+                ],
+                layout = (title = "Dash Data Visualization", barmode="group")
+            )
+        )
+    end
+
+    run_server(app, "0.0.0.0", 8000, debug=true)
```

Comment:
For parity with the instructions that follow below (using `127.0.0.1` instead of `0.0.0.0`):

```suggestion
    run_server(app, debug=true)
```